h2o-core/src/test/java/water/udf/DkvClassLoaderTest.java

```diff
@@ -20,15 +30,54 @@
 
   @Test
   public void testClassLoadFromKey() throws Exception {
-    Key k = loadTestJar("testKeyName.jar", "water/udf/cfunc_test.jar");
-    try {
-      ClassLoader cl = new DkvClassLoader(k, Thread.currentThread().getContextClassLoader());
-      Class clz = cl.loadClass("ai.h2o.TestRunnable");
-      Assert.assertNotNull(clz);
-      Runnable runnable = (Runnable) clz.newInstance();
-      runnable.run();
+    String testJar = "water/udf/cfunc_test.jar";
+    Key k = loadTestJar("testKeyName.jar", testJar);
+    ClassLoader cl = new DkvClassLoader(k, Thread.currentThread().getContextClassLoader());
+
+    int classCnt = 0;
```

Comment:
this test can be simplified just by calling

   assertLoadableResource(cl, "ai/h2o/TestRunnable.java", ...)
   assertLoadableResource(cl, "ai/h2o/TestRunnable.class")

Less complicated test is a better test ;)